### PR TITLE
sci-mathematics/gimps: version bump to version 29.8b6

### PR DESCRIPTION
**Pull request by PPed72**
Also bump to EAPI-7 and cleanup old version.
Closes: https://bugs.gentoo.org/694962
Signed-off-by: Paolo Pedroni <paolo.pedroni@iol.it>
Package-Manager: Portage-2.3.69, Repoman-2.3.16

### CODE REVIEW

File: sci-mathematics/gimps/gimps-29.8.6.ebuild [hunk 1]
```diff
@@ -1,36 +1,46 @@
...
+# The author has finally caved in to the pressure of putting the build
+# number in the package name. Some dark magic necessary to get to the
+# package name
+
+MY_PV=$(ver_rs 2 'b' )
+
 SRC_URI="
-	amd64? ( ftp://mersenne.org/gimps/p95v${PV/./}.linux64.tar.gz )
-	x86? ( ftp://mersenne.org/gimps/p95v${PV/./}.linux32.tar.gz )"
+	amd64? ( ftp://mersenne.org/gimps/p95v${MY_PV/./}.linux64.tar.gz )
```
Comment (juippis) [inline sci-mathematics/gimps/gimps-29.8.6.ebuild]: Could you rename the distfile into more identifiable one? Add `-> ${P}.linux64.tar.gz` or similar to this and line below. 

Also don't use `git mv` but make a new commit for addition and removal.

Comment (PPed72) [reply sci-mathematics/gimps/gimps-29.8.6.ebuild]: > Could you rename the distfile into more identifiable one? Add -> ${P}.linux64.tar.gz or similar to this and line below.

Is it really necessary? It's always been like this: p95 stands for Prime95, which was the original project name when it was a Windows only program, and it's been retained for historical reasons.

> Also don't use git mv but make a new commit for addition and removal.

I didn't.

I did a `git rm` of the old file, a `git add` of the new one and then `repoman dx commit`

I have no idea why it appeared as a `git mv`

Comment (juippis) [reply sci-mathematics/gimps/gimps-29.8.6.ebuild]: Ok, the distfile is good. But you need to make a new commit for new addition, then new commit for file removal.

Comment (PPed72) [reply sci-mathematics/gimps/gimps-29.8.6.ebuild]: Done. Is it ok now?

Comment (juippis) [reply sci-mathematics/gimps/gimps-29.8.6.ebuild]: Thanks! FYI, you can remove empty DEPEND from the ebuild if you wish to make it cleaner.

**Comment by PPed72** [comment]
In data lunedì 23 settembre 2019 15:37:15 CEST, juippis ha scritto:
> juippis commented on this pull request.
> Thanks! FYI, you can remove empty DEPEND from the ebuild if you wish to make
> it cleaner.

Next time ;)

Thanks for the attention and the mentoring.

-- 
Paolo Pedroni
Email: paolo.pedroni<at>iol.it

**Pull request closed without merging**